fjord/heartbeat/healthchecks.py

```diff
@@ -0,0 +1,205 @@
+from collections import Counter, namedtuple
+from datetime import datetime, timedelta
+import logging
+
+from django.conf import settings
+from django.core.mail import send_mail
+from django.db import connection
+
+from fjord.base.email_utils import render_email
+from fjord.heartbeat.models import Answer
+from fjord.mailinglist.utils import get_recipients
+
+
+log = logging.getLogger('i.heartbeat')
+
+
+SEVERITY_LOW = 1
+SEVERITY_MEDIUM = 5
+SEVERITY_HIGH = 10
+
+SEVERITY = {
+    SEVERITY_LOW: 'low',
+    SEVERITY_MEDIUM: 'medium',
+    SEVERITY_HIGH: 'high'
+}
+
+
+Result = namedtuple('Result', ['name', 'severity', 'summary', 'output'])
+
+
+CHECKS = []
+
+
+def register_check(cls):
+    CHECKS.append(cls)
+    return cls
+
+
+class Check(object):
+    name = ''
+
+    @classmethod
+    def check(cls):
+        pass
+
+
+@register_check
+class CheckAnyAnswers(Check):
+    """Are there any heartbeat answers? If not, that's very bad."""
+    name = 'any answers check'
```

Comment:
I think the question that you put in the docstring would be a great name as well, since the notification would end up saying "Are there any heartbeat answers? 0 answers in the last 24 hours." which is super-easy to understand.